modules/serverless-rn-1-14-0.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-14-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} 1.14.0
+
+{ServerlessProductName} 1.14.0 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1-14-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.20.0.
```

Comment:
I wonder whether a temporal qualifier like "now" is useful in release notes ? I probably would leave that out (everywhere)

Reply:
@rhuss for now I want to re-add these exactly as they were in the past to avoid any complications, unless there are changes that aren't just suggestions, grammatical, etc.
If you have general feedback (e.g. removing "now" going forward) please add it for the issue for the next version of the release notes (1.24.0) so that we don't need another change management process to approve the changes.
cc @rh-max